cmd/reconciler/main.go

```diff
@@ -131,6 +159,122 @@ func main() {
 		status.EnablePanicOnMisuse()
 	}
 
+	reconcilerScope := declared.Scope(*scope)
+	fight.SetFightThreshold(*fightDetectionThreshold)
+
+	cl, discoveryClient, baseApplier, supervisor, err := configureClients(reconcilerScope)
+	if err != nil {
+		klog.Error(err)
+	}
+
+	// Start listening to signals
+	signalCtx := signals.SetupSignalHandler()
+	mgrOptions := ctrl.Options{
```

Comment:
Do we want to set the MapperProvider here? Seems like we already have a discovery client and usually the mapper wraps that. But I'm not sure what the default mapper is.

Reply:
We might also want to set the Logger

Reply:
NewClient maybe too, if we already have a client.